.vscode/settings.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
I think you accidentally checked in your .vscode settings file

Reply:
whoopsies, fixing